README.md

```diff
@@ -24,3 +24,27 @@ Getting Started and full documentation can be found at [http://www.scala-native.
 
 Scala Native is distributed under the Apache License.
 [See LICENSE.md for details](https://github.com/scala-native/scala-native/blob/main/LICENSE.md)
+
+## Sponsors
+
+[<img src="https://www.epfl.ch/wp/5.5/wp-content/themes/wp-theme-2018/assets/svg/epfl-logo.svg?refresh=now" width=200>][EPFL-Link]
+
+[The École polytechnique fédérale de Lausanne (EPFL)][EPFL-Link] and [LAMP][EPFL_LAMP-Link] are founding development of Scala Native as a part Martin Odersky's research program of [Capatibilies for Resources and Effects (Caprese)][Caprese-Link] in Scala.
```

Comment:
```suggestion
[The École polytechnique fédérale de Lausanne (EPFL)][EPFL-Link] and [LAMP][EPFL_LAMP-Link] are funding the development of Scala Native as part of Martin Odersky's research program for [Capatibilies for Resources and Effects (Caprese)][Caprese-Link] in Scala.
```

Reply:
Capatibilities --> Capabilities
research program for --> research programme on